venafi/resource_venafi_certificate.go

```diff
@@ -28,107 +34,107 @@ func resourceVenafiCertificate() *schema.Resource {
 		Exists: resourceVenafiCertificateExists,
 
 		Schema: map[string]*schema.Schema{
-			"csr_origin": &schema.Schema{
```

Comment:
same as on the provider: why the "&schema.Schema" references are being removed on the resource attributes?